presentation/backend/src/presentation-backend/Constants.ts

```diff
@@ -7,7 +7,9 @@
  */
 
 import * as path from "path";
-import { PRESENTATION_COMMON_ROOT } from "@itwin/presentation-common";
+
+/** @internal */
+export const PRESENTATION_COMMON_ROOT = __dirname;
```

Comment:
This should have been left in `presentation-common` because now it points to `presentation-backend` root instead. I thought that only `getLocalesDirectory` was causing problems with webpack?

Reply:
I moved this to the backend, because that was the only place where it was used, I'm pretty sure we need a node polyfill to use `__dirname` in a common pkg. Thought its best to move it to the backend where we have access to node

Reply:
If I rememeber correctly the purpose of this constant is to know `presentation-common` package directory so we could know path to assets in that package.

@grigasp I see that in `presentation-common` assets folder we have only localization files. It looks like we would not need path to common assets if we dropped localization on backed?

Reply:
Yes, I agree with @saskliutas - the point of having `PRESENTATION_COMMON_ROOT` in `presentation-common` was to find assets of that package. I suppose this is still going to work if the backend is webpacked, but I'm pretty sure it won't when it's not.

And yes, we could **probably** stop doing all this if we stopped localizing stuff on the backend. We have a backlog item for this, but no time to work on it... Note: I say probably, because I don't have a solution at hand on how we're going to handle localization for backend-only apps.

Reply:
reverted this, was able to get it to work with leaving `PRESENTATION_COMMON_ROOT` as is; surprised this works, dont think `__dirname` is defined in the web; but guess removing the importh of `path` is all i needed to do.